packages/samples/react/src/components/table/stateful-with-selection.tsx

```diff
@@ -56,10 +61,11 @@ export const TableStatefulWithSelection: FC = () => {
 						[
 							{ key: 'id', label: '#ID', textAlign: 'left' },
 							{ key: 'name', label: 'Name', textAlign: 'left' },
+							{ key: 'action', label: 'Action', textAlign: 'left', render: (element) => renderButton(element) },
```

Comment:
```suggestion
							{ key: 'action', label: 'Action', textAlign: 'left', render: renderButton },
```

(Bitte auch die anderen Beispiele berücksichtigen)